vms/rpcchainvm/grpcutils/server.go

```diff
@@ -88,8 +89,8 @@ func WithStreamInterceptor(streamInterceptor grpc.StreamServerInterceptor) Serve
 
 // NewListener returns a TCP listener listening against the next available port
 // on the system bound to localhost.
-func NewListener() (net.Listener, error) {
-	return net.Listen("tcp", "127.0.0.1:")
+func NewListener(ctx context.Context) (net.Listener, error) {
+	return (&net.ListenConfig{}).Listen(ctx, "tcp", "127.0.0.1:")
```

Comment:
Because this function hardcodes localhost, it is unexpected for this `Listen` to ever block. I would prefer the prior behavior of passing in `context.Background()`.